README.md

```diff
@@ -1,6 +1,44 @@
 # Enarx
-A client for deploying WebAssembly applications into Enarx Keeps.
 
-This is currently a placeholder repository.
+## Introduction
+Enarx is an application deployment system enabling applications to run within Trusted Execution Environments (TEEs) without rewriting for particular platforms or SDKs. It handles attestation and delivery into a run-time “Keep” based on WebAssembly, offering developers a wide range of language choices for implementation. Enarx is CPU-architecture independent, enabling the same application code to be deployed across multiple targets, abstracting issues such as cross-compilation and differing attestation mechanisms between hardware vendors. Work is currently underway on AMD SEV and Intel SGX.
+
+We've known for a long time that we need encryption for data at rest and in transit: Enarx helps you do encryption for data in use.
```

Comment:
I'd suggest being a little less colloquial here. Perhaps:

```suggestion
Enarx aims to transparently enable encryption for code and data:
  * in transit
  * at rest
  * in use
```

Reply:
I don't think i agree here: i think we impart less context and don't make our intentions with Enarx as clear by just saying it aims to transparently enable encryption for code and data. Arguably, LUKS already does just that. Sure, it's not for code or data in use.
Which is why i think the context of encrypting data at rest, data in transit and data in use is helpful.

Adding "code and data" is useful however, and i've incorporated Stefan's suggestion.